CODE-OF-CONDUCT.md

```diff
@@ -59,8 +64,7 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+reported to the Istio Steering Committee at steering@discuss.istio.io.
```

Comment:
Does this discuss alias map back to the Istio steering committee google group?

Reply:
We might want to think about adding an explicit group/alias for folks to report code of conduct violations and add Steering committee as the only member for now. Keeping these separated will give an easier way to decouple Steering from this in future as our community grows and we create more dedicated smaller sub-groups.

Reply:
+1, creating a separate CoCC was one of the comments I added to the SC agenda document yesterday

Reply:
@ssuchter has the AI to create a CoC group and update the doc to reference it here.